performance/tests/Item20Items.test.js

```diff
@@ -0,0 +1,131 @@
+/* global page, maxCLS, stepNumber, testMultiple, maxDCL, maxFCP, maxLCP, passRatio */
+
+const TestResults = require('../TestResults');
+const {CLS, FID, FPS, getAverageFPS, PageLoadingMetrics} = require('../TraceModel');
+const {clsValue, firstInputValue, getFileName, scrollAtPoint} = require('../utils');
+
+describe('Item20Items', () => {
```

Comment:
I'm not sure, but if the test script is exactly the same except for the url, is there any way to import and reuse it?

Reply:
I managed to do some refactoring.
What I did was to put all the tests in `ListItemsTests.js` and export them then use them to run tests for `VirtualList`, `VirtualList100Items` and `Items`.
This is the leanest code I could come with. 
In `VirtualList.tests.js` I'm calling the tests for both normal VirtualList and VirtualList100Items` and in `Items` I'm calling the tests for 100 Items.
I also disabled ESLINT for `ListItemsTests.js` because of define errors thrown by `describe`, `it`, `expect` and so on. 
Let me know what you think about this approach.

Reply:
I love the your refactoring approach. It's nice to a lot of code being reused. Good job!.